core-services/src/main/java/org/zalando/nakadi/service/NakadiAvroSchemaService.java

```diff
@@ -0,0 +1,48 @@
+package org.zalando.nakadi.service;
+
+import org.apache.avro.Schema;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class NakadiAvroSchemaService {
+
+    public static final class VersionedAvroSchema {
```

Comment:
I suggest to move it to controller view, because it is clearly a view, which should not belong to service.